config/routes.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  resources :posts
+
+  get 'user/index'
+  get 'user/new'
+  get 'user/show'
+  get 'user/create'
```

Comment:
Please remove these 4 lines as you have already defined necessary routes for users below